runtime-config-linux.md

```diff
@@ -144,19 +144,29 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
 
-The path to the cgroups can be specified in the Spec via `cgroupsPath`.
-`cgroupsPath` is expected to be relative to the cgroups mount point.
-If `cgroupsPath` is not specified, implementations can define the default cgroup path.
+The name of the cgroups can be specified in the Spec via `cgroupsName`.
+Cgroup names mimic filesystem paths closely since they express a hierarchy of cgroups.
+`cgroupsName` is expected to be absolute.
+An absolute name is one that is defined from the root cgroup `/`.
+For example, `/foo/bar` is acceptible. `foo/bar` is not acceptible.
+Allowable characters for cgroup names are,
+Alpha numeric ([a-zA-Z0-9]+)
+Underscores (_)
+Dashes (-)
+Periods (.)
+In general and unless otherwise specified, regular filesystem path rules apply.
```

Comment:
On Mon, Dec 28, 2015 at 11:56:49AM -0800, Vish Kannan wrote:
> > +In general and unless otherwise specified, regular filesystem path rules apply.
> 
> In terms of characters, only the ones mentioned above are
> supported. But path construction follow filesystem path rules.
> @hqhq: What link do you need?

I can't find a reference to this in the Linux kernel docs, but POSIX
says pretty clearly that everything except the null and slash bytes
are fair game as long as the full byte name isn't ‘.’ or ‘..’ [1].

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_170